### PR TITLE
add new items from latest master.mdb

### PR DESCRIPTION
**Pull request by stardustichi**

### CODE REVIEW

File: src/data/trainer-title-requirements.csv [hunk 1]
```diff
@@ -177,4 +177,8 @@
 "ビワハヤヒデのファン数を\n累計1000万人獲得しよう","Reach 10 million total fans for Biwa Hayahide"
 "ビワハヤヒデのファン数を\n累計5000万人獲得しよう","Reach 50 million total fans for Biwa Hayahide"
 "ビワハヤヒデのファン数を\n累計1億人獲得しよう","Reach 100 million total fans for Biwa Hayahide"
+"カレンチャンの\nウマ娘ストーリー第4話を見よう","Read chapter 4 of Currenchan's story"
```
Comment (FabulousCupcake) [inline src/data/trainer-title-requirements.csv:180]:
```suggestion
"カレンチャンの\nウマ娘ストーリー第4話を見よう","Read chapter 4 of Curren Chan's story"
```

**Review by FabulousCupcake — CHANGES_REQUESTED** [review]

**Comment by stardustichi** [comment]
hmm i think there are some new one in missions too

**Comment by FabulousCupcake** [comment]
Yes; the procedure for when there's a new (master.mdb) update is to run `scripts/import-new-mdb.sh`.  
This will capture all the new empty fields and add it to (almost) all csv files.

I will run it after this pr is merged.

**Review by FabulousCupcake — APPROVED** [review]

**Comment by FabulousCupcake** [comment]
Thank you!

**Pull request merged**